### PR TITLE
[bitnami/fluentd] add means to configurate liveness and readiness endpoints for forwarder and aggregator

### PR DESCRIPTION
**Pull request by ekrucio**
**Description of the change**

Added means to additionally configure the path and port for liveness and readiness probe in both aggregator and forwarder

**Benefits**

Before this change there were no ways to configure the endpoints to which the probes should run. This was unnecessary difficult to work with if you're trying to add fluentd to an already existing monitoring setup. The default values should preserve the current behaiviour.

**Possible drawbacks**

None for what I know.

**Applicable issues**

None

**Additional information**

This change has been tested with https://hub.docker.com/r/bitnami/fluentd/latest and no issues were encountered.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/fluentd/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: fluentd
 sources:
   - https://github.com/bitnami/bitnami-docker-fluentd
   - https://www.fluentd.org/
-version: 3.2.2
+version: 3.2.3
```
Comment (alemorcuq) [inline bitnami/fluentd/Chart.yaml:28]:
```suggestion
version: 3.3.0
```

File: bitnami/fluentd/values-production.yaml [hunk 2]
```diff
@@ -162,13 +162,19 @@ forwarder:
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
   ##
   livenessProbe:
+    httpGet:
+      path: /fluentd.healthcheck?json=%7B%22ping%22%3A+%22pong%22%7D
+      port: http
     enabled: true
```
Comment (alemorcuq) [inline bitnami/fluentd/values-production.yaml:168]: Please, let `enabled: true` be the first setting (this applies to all blocks)

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR, @ekrucio, it's a nice addition. I've added some minor comments to the PR.

**Comment by ekrucio** [comment]
@alemorcuq I've fixed the comments you mentioned. It's better this way, I can agree. 👍

**Review by alemorcuq — APPROVED** [review]
Thanks!

**Pull request merged**